coala_quickstart/generation/Settings.py

```diff
@@ -61,7 +61,7 @@ def generate_ignore_field(project_dir, languages, extset, ignore_globs):
     for glob in ignore_globs:
         gitignore_files = {file
                            for file in collect_files([glob], null_printer)}
-        if not all_files.isdisjoint(gitignore_files):
+        if all_files.isdisjoint(gitignore_files):
```

Comment:
or alternatively, just remove the ignore field altogether

```py
all_files = set(collect_files(
        "**",
        null_printer,
        ignored_file_paths=ignore_globs))
```
and `all_files` now should have all the ignore_files already removed (expectedly) and we can use it to set the `files` setting value for the default section.